src/main/kotlin/org/opensearch/commons/alerting/model/ScheduledJob.kt

```diff
@@ -0,0 +1,85 @@
+package org.opensearch.commons.alerting.model
+
+import org.opensearch.common.xcontent.ToXContent
+import org.opensearch.common.xcontent.XContentBuilder
+import org.opensearch.common.xcontent.XContentParser
+import org.opensearch.common.xcontent.XContentParserUtils
+import org.opensearch.commons.utils.BaseModel
+import java.io.IOException
+import java.time.Instant
+
+interface ScheduledJob : BaseModel {
```

Comment:
Similar to previous comment, looks like some of these could be avoided if we could segregate code and structure in the monitor model